### ISSUE TITLE
Docs for WaterBridgeAnalysis incorrectly formatted/do not show up

### ISSUE THREAD

**Issue by orbeckst** [MDAnalysis/mdanalysis#3259]
## Expected behavior ##

WaterBridgeAnalysis https://docs.mdanalysis.org/2.0.0-dev0/documentation_pages/analysis/wbridge_analysis.html shows up as its own entry.

## Actual behavior ##

Instead it shows up under https://docs.mdanalysis.org/2.0.0-dev0/documentation_pages/analysis_modules.html#hydrogen-bonding

## Code to reproduce the behavior ##

See links above.

This looks like an issue with the title formatting.

## Current version of MDAnalysis ##

- Which version are you using? (run `python -c "import MDAnalysis as mda; print(mda.__version__)"`) 2.0.0-dev docs

**Comment by xiki-tempula** [comment]
Thanks for pointing out this issue. I think I might need some help in this case.
Previously, I set the title as

```
Water Bridge Analysis --- :mod:`MDAnalysis.analysis.hydrogenbonds.wbridge_analysis`
===================================================================================
```

Which gives the correct doc but would give PEP warning as the line is longer than the permitted length.
I blindly change this to 

```
Water Bridge Analysis 
--- :mod:`MDAnalysis.analysis.hydrogenbonds.wbridge_analysis`
=============================================================
```
which give rise to this issue.
I wonder if there is any way of separating this line into two lines?

**Comment by xiki-tempula** [comment]
I think I might need some help with regard to this issue.

Previously, the title is

```
Water Bridge Analysis --- :mod:`MDAnalysis.analysis.hydrogenbonds.wbridge_analysis`
=================================================================================== 
```

However, I got PEP warning and I blindly change it to 

```
Water Bridge Analysis --- \
:mod:`MDAnalysis.analysis.hydrogenbonds.wbridge_analysis`
=========================================================
```
which seems to give this problem. I wonder if there is any way of splitting the title into two lines? Thanks.

**Comment by orbeckst** [comment]
Ignore the PEP8 warning. 

Check the docs when they are built. They appear on RTD and I think they get linked in the test report.

**Comment by orbeckst** [comment]
I don't know of a way to split the line – maybe the sphinx or reST docs have something. Or maybe @lilyminium knows of a trick?

**Comment by IAlibay** [comment]
I'm adding this as a 2.0 milestone, I guess if it's just a single line change it can just be done in #3120 ?

**Comment by orbeckst** [comment]
This will get fixed as part of PR #3258
